Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/DriverOnboarding/VehicleRegistrationCertificate.hs

```diff
@@ -281,6 +290,74 @@ compareRegistrationDates actualDate providedDate =
   isJust providedDate
     && ((convertUTCTimetoDate <$> providedDate) /= (convertUTCTimetoDate <$> convertTextToUTC actualDate))
 
+onVerifyRCWithGovtData :: Verification.GovtDataResponse -> Person.Person -> Id Image.Image -> Maybe Vehicle.Variant -> Flow ()
+onVerifyRCWithGovtData govtDataRes person documentImageId1 dashboardPassedVehicleVariant = do
+  now <- getCurrentTime
```

Comment:
Why do we need to handle output of GovtData and Idfy differently? Ideally, from the verificationService itself output of both should come similar.